### ISSUE TITLE
Trace server started externally cannot be stopped

### ISSUE THREAD

**Issue by tahini** [theia-ide/theia-trace-extension#252]
And yet, the "Stop trace server" button is available to click on. Clicking on it writes a the following message: "Failed to stop the trace server on port: 8080.", without further reason.

Either do not allow to click on the "stop trace server" and just replace by a "Trace server running" status, or else, write in the error message that the server was not started by the application and it cannot be stopped this way.

**Comment by bhufmann** [comment]
Yes, external started server cannot be stopped anymore. In the recent updates for start/stop the trace server is stopped by using npm package tree-kill using the PID that was obtained when starting the server by the extension's Theia back-end code. Before the back-end was deleting the process that was holding the port configured in the preferences which could be the trace server process or not. Also, tree kill is platform independent that previous implementation.

However, the current start/stop procedure for the local needs as well as the connection monitoring to be revised in the future.
